includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -945,12 +945,15 @@ private function update_woopay_enabled_locations( WP_REST_Request $request ) {
 			return;
 		}
 
-		$woopay_enabled_locations = $request->get_param( 'woopay_enabled_locations' );
+		$wcpay_form_fields = $this->wcpay_gateway->get_form_fields();
 
-		$all_locations = $this->wcpay_gateway->form_fields['payment_request_button_locations']['options'];
-		WC_Payments::woopay_tracker()->woopay_locations_updated( $all_locations, $woopay_enabled_locations );
+		if ( isset( $wcpay_form_fields['payment_request_button_locations']['options'] ) ) {
```

Comment:
In this case, I'd be afraid this `if` statement could silence the original issue and create a different problem where the enabled locations settings are not updated.

I'd prefer only using `get_form_fields()` here as well, as we're not 100% sure what's causing the error, although I understand the intentional defensive programming.

Reply:
As discussed in Slack, I reverted the added `if` since there are other calls to `get_form_fields` in the same file. Done in d99cace.